config/initializers/gds-api-adapters.rb

```diff
@@ -0,0 +1 @@
+GdsApi.config.always_raise_for_not_found = true
```

Comment:
In other apps, we set the following as well:
```
  # Return a hash, not an OpenStruct from a request.
  config.hash_response_for_requests = true
```

Do we not need that in this app?